### ISSUE TITLE
Blank screen while trying open settings

### ISSUE THREAD

**Issue by Nasheedabdul** [brave/brave-browser#12199]
## Description 
Can't use menu of brave and everything is blank

## Steps to reproduce 

   1. Screen blank
   2. Issue in dark mode appearance
   3. 

## Actual result 
![Screenshot_20201019_232427_com brave browser](https://user-images.githubusercontent.com/68918787/96495099-f74d4d00-1264-11eb-8190-03f84e066811.jpg)
![Screenshot_20201019_232638_com brave browser](https://user-images.githubusercontent.com/68918787/96495106-fa483d80-1264-11eb-97df-75bd7ca22161.jpg)

## Expected result

Should open menu in dark mode also
## Version/Channel Information:

Specific channel 
- Can you reproduce this issue with the current Play Store version? Yes
- Can you reproduce this issue with the current Play Store Beta version? Don't know
- Can you reproduce this issue with the current Play Store Nightly version?don't know 

## Device details
   - Install type (ARM, x86):ARM
   - Device type (Phone, Tablet, Phablet):phone
   - Android version:9
   
## Brave version

1.15.75
### Website problems only
- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards?No
- Is the issue reproducible on the latest version of Chrome? Didn't try

**Comment by hollons** [comment]
@Nasheedabdul thanks for reporting. Does the issue continue if you restart your phone?

**Comment by Nasheedabdul** [comment]
@hollons Problem continues after restart

**Comment by Nasheedabdul** [comment]
@SergeyZhukovsky

**Comment by Nasheedabdul** [comment]
While changing appearance to white then issue gets solved

**Comment by hollons** [comment]
@Nasheedabdul removed the custom font but issue persisted.

**Comment by SergeyZhukovsky** [comment]
@Nasheedabdul do you still experience that issue?

**Comment by Nasheedabdul** [comment]
No.now i am experiencing hang while using multiple tab in brave
